util/lang_def/restest_yacc.py

```diff
@@ -0,0 +1,63 @@
+import ply.yacc as yacc
+from util.lang_def.restest_lexer import Lexer
+
+tokens = Lexer.tokens
+lexer = Lexer().build()
+
```

Comment:
Can the parser be wrapped in a class? I'm not really sure if it is possible though i'll have to make some research and get back to you.

Reply:
I think it can, and it's probably a better idea to do it that way. I'll make the changes.